Extension/src/LanguageServer/client.ts

```diff
@@ -2964,6 +2964,16 @@ export class DefaultClient implements Client {
         this.handleRemoveCodeAnalysisProblems(false, identifiersAndUris);
     }
 
+    private static countNewlines(text: string): number {
```

Comment:
Not sure that this needs to be a static function in a class.  It could just be a regular function outside a class (e.g. a "util" function).